espanol/La_ultima_orden/the_last_order.md

```diff
@@ -0,0 +1,14 @@
+La Ultima Orden
+===============
+
+*Arthur C. Clarke*
+
+-...Os habla el presidente. El hecho de que estéis oyendo este mensaje significa que ya he muerto y que nuestro país ha sido destruido. Pero vosotros sois soldados... sois los más adiestrados de toda nuestra historia. Vosotros sabéis obedecer órdenes. Ahora tenéis que obedecer la más dura que jamás habéis recibido... ¿Dura? Pensó el primer oficial de radar amargamente. No; ahora sería fácil, dado que habían visto la tierra que amaban abrasada por el fuego de multitud de soles. Ya no cabían las vacilaciones ni los escrúpulos de que la venganza de los dioses cayera igualmente sobre inocentes y culpables. Pero, ¿por qué, por qué se había dejado para tan tarde?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.